src/capi/property.cpp

```diff
@@ -97,8 +98,3 @@ extern "C" chfl_status chfl_property_get_vector3d(const CHFL_PROPERTY* const pro
         value[2] = vector[2];
     )
 }
-
-extern "C" chfl_status chfl_property_free(const CHFL_PROPERTY* property) {
```

Comment:
Why doesn't `chfl_free` return `CHFL_SUCCESS`? Is it to remove clutter?

Reply:
Yes, and to match the signature of `free`